lib/models/file-info.js

```diff
@@ -95,7 +97,33 @@ var FileInfo = CoreObject.extend({
     this._fileContents = fse.readFileSync(fullPath, { encoding: 'utf8' });
   },
 
+  updateImports: function() {
+    if (this.ext !== '.js' || !this._fileContents) { return; } // only process JavaScript files
+
+    var appName = this.options.projectName;
+
+    try {
+      var newContents = importDeclarationsTransform(
+        { source: this._fileContents,
+          fileInfo: this,
+          appName: appName,
+          fileInfos: this._fileInfoCollection._fileInfos
+        },
+        { jscodeshift: jscodeshift });
```

Comment:
ok, but just for the code changes included in this PR. For the rest of the code, I'd update it in another commit / PR